programs/multicall-handler/src/lib.rs

```diff
@@ -12,9 +12,9 @@ declare_id!("6kqWTz3A3ZYMV2FMU24ke8rHzT82SaBz7GkBKTd7Z9BH");
 pub mod multicall_handler {
     use super::*;
 
-    // Handler to receive AcrossV3 message formatted as serialized message compiled instructions. When deserialized,
+    // Handler to receive Across message formatted as serialized message compiled instructions. When deserialized,
     // these are matched with the passed accounts and executed as CPIs.
-    pub fn handle_v3_across_message(ctx: Context<HandleV3AcrossMessage>, message: Vec<u8>) -> Result<()> {
+    pub fn handle_v3_across_message(ctx: Context<HandleAcrossMessage>, message: Vec<u8>) -> Result<()> {
```

Comment:
we do tend to name contexts same as camelcased function name, so this would not be consistent

Reply:
true. fixed